dandiapi/api/views/dandiset.py

```diff
@@ -166,6 +181,14 @@ def create(self, request):
                     'includeInCitation': True,
                 },
             ],
+            'access': [
+                {
+                    'schemaKey': 'AccessRequirements',
+                    'status': 'dandi:OpenAccess'
+                    if embargo_status == Dandiset.EmbargoStatus.OPEN
+                    else 'dandi:EmbargoedAccess',
+                }
+            ],
```

Comment:
This is schema-dependant and should be generated by a function exported by dandischema I think, or at the minimum we should have an issue/TODO to do eventually move it there.

Reply:
https://github.com/dandi/dandi-api/issues/666